### ISSUE TITLE
Definition of F in script C

### ISSUE THREAD

**Issue by ghost** [contactmodel/COVID19-Japan-Reff#7]
`F` is defined in the stan code in `C. Calculating the Rt in Stan.ipynb` as follows:

>  for (k in 1:upper_bound) 
 F_tmp[k] = ReportDelay[upper_bound+1-k];
        F = head(F_tmp, K);

 `F` is the delay time from infection to report, Prof. Nishiura said in the lecture on 5/12.
Shouldn't `conv_report_inc` be used instead of `ReportDelay` in this part?

**Comment by aakhmetz** [comment]
Yes, that's absolutely correct - Thank you for noticing! It's a strange copy-paste typo, because I have had the correct version in my original program adapted purely to cmdStan. I will correct it in today's update
